### PR TITLE
Fixed resetting of physics objects.

### PR DESCRIPTION
**Pull request by BeeHiveJava**

### CODE REVIEW

File: Soccar [Game-Server]/src/nl/soccar/gameserver/controller/socnet/message/EventMessage.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ *
+ * @author PTS34A
+ */
+@MessageEvent(id = MessageConstants.EVENT_MESSAGE, handler = EventMessageHandler.class)
+public final class EventMessage extends Message {
+
+    private final EventType type;
+    private final int playerId;
+
+    public EventMessage(EventType type, int playerId) {
```
Comment (TehMessiah) [inline Soccar [Game-Server]/src/nl/soccar/gameserver/controller/socnet/message/EventMessage.java]: MINOR Document this public constructor. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)

**Comment by TehMessiah** [comment]
SonarQube analysis reported 2 issues
* MAJOR 1 major
* MINOR 1 minor

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [GameWrapper.java#L66](https://github.com/PTS3-S34A/Game-Server/blob/bb9b5ec21e49c1012b0fe109a5b52b13c0b42e5b/Soccar [Game-Server]/src/nl/soccar/gameserver/model/session/GameWrapper.java#L66): Reduce this anonymous class number of lines from 24 to at most 20, or make it a named class. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1188)

**Comment by TehMessiah** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [GameWrapper.java#L66](https://github.com/PTS3-S34A/Game-Server/blob/addb53d921767ced718aed96e3e90224deffe6e3/Soccar [Game-Server]/src/nl/soccar/gameserver/model/session/GameWrapper.java#L66): Reduce this anonymous class number of lines from 24 to at most 20, or make it a named class. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1188)

File: Soccar [Game-Server]/src/nl/soccar/gameserver/controller/socnet/message/EventMessage.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import nl.soccar.socnet.message.Message;
+import nl.soccar.socnet.message.MessageConstants;
+import nl.soccar.socnet.message.MessageEvent;
+
+/**
+ * Message for changing host.
+ *
+ * @author PTS34A
+ */
+@MessageEvent(id = MessageConstants.EVENT_MESSAGE, handler = EventMessageHandler.class)
```
Comment (Ranray) [inline Soccar [Game-Server]/src/nl/soccar/gameserver/controller/socnet/message/EventMessage.java]: MessageConstants moet EVENT_MESSAGE_ID zijn.

File: Soccar [Game-Server]/src/nl/soccar/gameserver/model/session/ServerGameEventListener.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+import nl.soccar.gameserver.model.PlayerWrapper;
+import nl.soccar.library.*;
+import nl.soccar.library.enumeration.EventType;
+import nl.soccar.physics.GameEngine;
+import nl.soccar.physics.listener.GameEventListener;
+
+import java.time.LocalTime;
+
+/**
+ * @author PTS34A
```
Comment (Ranray) [inline Soccar [Game-Server]/src/nl/soccar/gameserver/model/session/ServerGameEventListener.java:16]: Documentatie

**Review by Ranray — CHANGES_REQUESTED** [review]

**Comment by TehMessiah** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [GameWrapper.java#L66](https://github.com/PTS3-S34A/Game-Server/blob/a21f581b31c3ea7db2955528b9999ef9cac1a84b/Soccar [Game-Server]/src/nl/soccar/gameserver/model/session/GameWrapper.java#L66): Reduce this anonymous class number of lines from 24 to at most 20, or make it a named class. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1188)

**Comment by ghost** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [GameWrapper.java#L66](https://github.com/PTS3-S34A/Game-Server/blob/3c6b84d1a697d80cb3d1b1a28ea53286d7ecadfe/Soccar [Game-Server]/src/nl/soccar/gameserver/model/session/GameWrapper.java#L66): Reduce this anonymous class number of lines from 24 to at most 20, or make it a named class. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1188)

**Review by Ranray — APPROVED** [review]

**Pull request merged**